contracts/cw-core/src/state.rs

```diff
@@ -22,6 +22,7 @@ pub struct Config {
     pub automatically_add_cw721s: bool,
 }
 
+pub const ADMIN: Item<Addr> = Item::new("admin");
```

Comment:
I normally prefer to use `Item<Option<Addr>>` in these cases. Seems slightly less error prone as it forces the programmer to recon with the type being optional. Otherwise, might be nice to have a comment here explaining that this field will not always be set.

Reply:
Done.